### PR TITLE
Scaffold connectivity-adapter component

### PR DESCRIPTION
**Pull request by aszecowka**
**Description**

Changes proposed in this pull request:

- ...
- ...
- ...

**Related issue(s)**

### CODE REVIEW

File: components/connectivity-adapter/README.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# Connectivity Adapter
+
+Connectivity Adapter is responsible for translating [Kyma Connector Service API](https://kyma-project.io/docs/1.9/components/application-connector/specifications/connectorapi/)
```
Comment (crabtree) [inline components/connectivity-adapter/README.md]: I think those links should point to the master/latest documentation.

File: components/connectivity-adapter/README.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+# Connectivity Adapter
+
+Connectivity Adapter is responsible for translating [Kyma Connector Service API](https://kyma-project.io/docs/master/components/application-connector/specifications/connectorapi/)
```
Comment (klaudiagrz) [inline components/connectivity-adapter/README.md]:
```suggestion
Connectivity Adapter translates [Kyma Connector Service API](https://kyma-project.io/docs/master/components/application-connector/specifications/connectorapi/) 
```

**Pull request merged**